practical-unit-testing-with-rtl/Exercise1-queries/solution/src/index copy.js

```diff
@@ -0,0 +1,15 @@
+import { StrictMode } from "react";
+import { createRoot } from "react-dom/client";
+
+import App from "./App";
+
+if (process.env.NODE_ENV === "development") {
+	const rootElement = document.getElementById("root");
+	const root = createRoot(rootElement);
+
+	root.render(
+		<StrictMode>
+			<App />
+		</StrictMode>
+	);
+}
```

Comment:
Is this `index copy.js` file intentional?

Reply:
🤦  dont push PRs before lunch lol